reference/excel/chartformatstring.md

```diff
@@ -9,7 +9,7 @@ None
 ## Relationships
 | Relationship | Type	|Description| Req. Set|
 |:---------------|:--------|:----------|:----|
-|font|[ChartFont](chartfont.md)|Represents the font properties of chartFormatString object. Read-only.|[1.9](../requirement-sets/excel-api-requirement-sets.md)|
+|font|[ChartFont](chartfont.md)|Represents the font attributes, such as font name, font size, color, etc. of chart characters object. Read-only.|[1.9](../requirement-sets/excel-api-requirement-sets.md)|
```

Comment:
Please remove the "etc." here and list all the font attributes, or make it clear that these are examples.